static/app/views/eventsV2/table/quickContext.tsx

```diff
@@ -33,49 +40,138 @@ export function hasContext(
-  const [data, setData] = useState<{msg: string} | null>(null);
+  const [data, setData] = useState<IssueData | null>(null);
+
+  // NOTE: Will extend when we add more type of contexts.
 
   useEffect(() => {
-    const mockData = {msg: t('Displaying Context for issue.')};
+    // Track mounted state so we dont call setState on unmounted components
+    let unmounted = false;
+
+    fetchData(api, props.dataRow, props.column)
+      .then(response => {
+        if (unmounted) {
+          return;
+        }
 
-    // Will replace with issue data.
-    const timer = setTimeout(() => {
-      setData(mockData);
-      setLoading(false);
-    }, 500);
+        setData(response);
+      })
+      .catch(() => {
+        setError(true);
+      })
+      .finally(() => {
+        setLoading(false);
+      });
 
     return () => {
-      clearTimeout(timer);
+      // If component has unmounted, dont set state
+      unmounted = true;
     };
-  }, []);
+  }, [api, props.dataRow, props.column]);
 
   return (
-    <ContextContainer>
+    <Wrapper>
       {loading ? (
         <NoContextWrapper>
           <LoadingIndicator hideMessage size={32} />
         </NoContextWrapper>
       ) : error ? (
         <NoContextWrapper>{t('Failed to load context for column.')}</NoContextWrapper>
       ) : isIssueContext(props.dataRow, props.column) && data ? (
-        <NoContextWrapper>{data.msg}</NoContextWrapper>
+        <IssueContext data={data} />
       ) : (
         <NoContextWrapper>{t('There is no context available.')}</NoContextWrapper>
       )}
+    </Wrapper>
+  );
+}
+
+// Only includes issue status context for now.
+function IssueContext(props: {data: IssueData}) {
+  const statusTitle = t('Issue Status');
+  const {status} = props.data;
+
+  return (
+    <ContextContainer>
+      <ContextTitle>{statusTitle}</ContextTitle>
+      <ContextBody>
+        {status === 'ignored' ? (
+          <IconMute data-test-id="quick-context-mute-icon" color="gray500" size="sm" />
+        ) : status === 'resolved' ? (
+          <IconCheckmark color="gray500" size="sm" />
+        ) : (
+          <IconNot data-test-id="quick-context-not-icon" color="gray500" size="sm" />
+        )}
+        <StatusText>{status}</StatusText>
+      </ContextBody>
     </ContextContainer>
   );
 }
 
 const ContextContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  margin: ${space(1.5)};
+
+  & + & {
+    margin-top: ${space(1.5)};
+  }
+`;
+
+const ContextTitle = styled('h6')`
+  color: ${p => p.theme.subText};
+  display: flex;
+  font-size: ${p => p.theme.fontSizeMedium};
+  margin: 0;
+`;
+
+const ContextBody = styled('div')`
+  margin: ${space(1.5)} 0 0;
+  width: 100%;
+  text-align: left;
+  font-size: ${p => p.theme.fontSizeLarge};
+  display: flex;
+  align-items: center;
+`;
+
+const StatusText = styled('span')`
+  margin-left: ${space(1)};
+  text-transform: capitalize;
+  color: 'purple400';
```

Comment:
You don't need the surrounding quotes here, but I was wondering if you intend for the text to have the purple color? I subbed in `${p => p.theme.purple400}` because that's how we usually pull theme values and I think it looks better without this

Reply:
Will fix this.

Reply:
Addressed in the latest commit. Color is not needed there at all.